### PR TITLE
Sagecrm

### LINKED ISSUE #545: New Element: Sage CRM
## Description
New Element documentation for Sage CRM Element

### PR DESCRIPTION
**Pull request by davewilks**
## Highlights
* New element

## Closes
* Closes #545

### CODE REVIEW

File: docs/elements/sagecrm/authenticate.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+
+    ```json
+    {
+      "element": {
+        "key": "{{page.elementKey}}"
+      },
+      "configuration": {
+         "subdomain": "sagecrm-0.<YOUR_TENANT>.com",
+        	"username": "<YOUR_USERNAME>",
```
Comment (claytonshaver) [inline docs/elements/sagecrm/authenticate.md]: We should probably fix the spacing on this.

**Comment by davewilks** [comment]
@claytonshaver I fixed the spacing and also fixed the spacing in another block of sample code.

**Comment by claytonshaver** [comment]
Hey @davewilks do you want me to resolve the conflicts and merge it now? Or is this one on hold?

**Comment by davewilks** [comment]
Please do. Thanks.

**Pull request merged**